packages/api-gateway/src/routers/apiGatewayRouter.ts

```diff
@@ -341,7 +348,21 @@ const apiGatewayRouter = (
     .get(
       "/organizations/:organizationId",
       authorizationMiddleware([M2M_ROLE]),
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        const ctx = fromApiGatewayAppContext(req.ctx, req.headers);
+
+        try {
+          const organization = await tenantService.getOrganization(
+            ctx,
+            req.params.organizationId
+          );
+
+          return res.status(200).json(organization).send();
+        } catch (error) {
+          const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
+          return res.status(errorRes.status).json(errorRes).end();
+        }
+      }
     )
     .post(
       "/organizations/origin/:origin/externalId/:externalId/attributes/:code",
```

Comment:
Not implemented as part of this PR, need to clarify how to call partyRegistryProxyService https://github.com/pagopa/interop-be-api-gateway/blob/e2de8b7a3265da4cd1b41424f59594fd9bfda2e3/src/main/scala/it/pagopa/interop/apigateway/api/impl/GatewayApiServiceImpl.scala#L97

Reply:
Implemented in [e7623ef](https://github.com/pagopa/interop-be-monorepo/pull/914/commits/e7623efb25a8a4f7a4593d4f80ce4bee2eb7d669) after clarifying open points (no need to call party registry proxy anymore)